src/models/connectionInfo.ts

```diff
@@ -167,10 +179,11 @@ export function getUserNameOrDomainLogin(creds: Interfaces.IConnectionCredential
  */
 export function getTooltip(connCreds: Interfaces.IConnectionCredentials, serverInfo?: ConnectionContracts.ServerInfo): string {
     let tooltip: string =
-           'Server name: ' + connCreds.server + '\r\n' +
+           connCreds.connectionString ? 'Connection string: ' + connCreds.connectionString + '\r\n' :
```

Comment:
Looks fine on my Mac. Maybe Windows requires it and it doesn't make a difference on other platforms?